about.html

```diff
@@ -83,7 +83,7 @@ <h4>Blockckhain world Conference logo 2022</h4>
         <section class="past">
```

Comment:
The section right below is missing the divider below the header.

<img width="991" alt="Screen Shot 2022-07-27 at 9 21 51" src="https://user-images.githubusercontent.com/82426682/181271471-033dd1ce-fa0e-4274-8ee2-f66ebdc4080b.png">